src/MainWindow/main_window.cpp

```diff
@@ -284,7 +319,28 @@ void MainWindow::createProgressBar() {
   m_progressWidget->hide();
 }
 
+void MainWindow::createRecentMenu() {
+  for (auto it{m_recentProjectsActions.begin()};
+       it != m_recentProjectsActions.end(); ++it)
+    delete it->first;
+  m_recentProjectsActions.clear();
+
+  for (int projCounter = 9; projCounter >= 0; projCounter--) {
```

Comment:
This part seems to be left out from re-factoring.

Reply:
Done